app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,24 @@
+class CommentsController < ApplicationController
+  def new
+    @comment = Comment.new
+  end
+
+  def create
+    @comment = Comment.new(comment_params)
+    @comment.user_id = params[:user_id]
+    @comment.post_id = params[:post_id]
+    @post = Post.find(params[:post_id])
```

Comment:
- [ ] It will be great to ensure that the comment is created on behalf of the currently logged in user (current_user) of the app. what you are doing here is that your getting your user from the params. This should not be the case. You should get your user from the `current_user` method that you were supposed to create in the application controller